share/completions/pnpm.fish

```diff
@@ -0,0 +1,20 @@
+function _pnpm_completion
+  set cmd (commandline -o)
+  set cursor (commandline -C)
+  set words (count $cmd)
+
+  set completions (eval env DEBUG=\"" \"" COMP_CWORD=\""$words\"" COMP_LINE=\""$cmd \"" COMP_POINT=\""$cursor\"" pnpm completion -- $cmd)
+
+  if [ "$completions" = "__tabtab_complete_files__" ]
```

Comment:
Where do you get this from? It's not in the original script?

Reply:
These conditions are based on the following.
https://github.com/pnpm/tabtab/commit/63aef480af1a0aee23f82e86cc167a6d2dc22eac